### PR TITLE
zh-cn(update): sync worker-src for the English

### PR DESCRIPTION
**Pull request by fwqaaq**
* part of #3893

### CODE REVIEW

File: files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md [hunk 1]
```diff
@@ -5,56 +5,74 @@ slug: Web/HTTP/Headers/Content-Security-Policy/worker-src
...
+请注意，这套相同的值可以用于所有 {{Glossary("fetch directive", "fetch 指令")}}（以及[许多其它的指令](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy/Sources#relevant_directives)）。
 
-## Examples
+## 示例
 
-### Violation cases
+### 违规的案例
 
-Given this CSP header:
+给定此 CSP 以下的 header：
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md:50]:
```suggestion
给定此 CSP header：
```

File: files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md [hunk 2]
```diff
@@ -5,56 +5,74 @@ slug: Web/HTTP/Headers/Content-Security-Policy/worker-src
...
-| Directive type | {{Glossary("Fetch directive")}}                                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| Fallback       | If this directive is absent, the user agent will first look for the {{CSP("child-src")}} directive, then the {{CSP("script-src")}} directive, then finally for the {{CSP("default-src")}} directive, when governing worker execution.Chrome 59 and higher skips the {{CSP("child-src")}} directive.Edge 17 skips the {{CSP("script-src")}} directive ([bug](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/17415478/)). |
-
-## Syntax
-
-One or more sources can be allowed for the `worker-src` policy:
-
-```plain
+HTTP {{HTTPHeader("Content-Security-Policy", "内容安全策略")}}（CSP）**`worker-src`** 指令指定了 {{domxref("Worker")}}、{{domxref("SharedWorker")}} 或 {{domxref("ServiceWorker")}} 脚本的有效来源。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md:8]:
```suggestion
HTTP {{HTTPHeader("Content-Security-Policy")}}（内容安全策略，CSP）**`worker-src`** 指令指定了 {{domxref("Worker")}}、{{domxref("SharedWorker")}} 或 {{domxref("ServiceWorker")}} 脚本的有效来源。
```

File: files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md [hunk 3]
```diff
@@ -5,56 +5,74 @@ slug: Web/HTTP/Headers/Content-Security-Policy/worker-src
...
+请注意，这套相同的值可以用于所有 {{Glossary("fetch directive", "fetch 指令")}}（以及[许多其它的指令](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy/Sources#relevant_directives)）。
 
-## Examples
+## 示例
 
-### Violation cases
+### 违规的案例
 
-Given this CSP header:
+给定此 CSP header：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md:50]:
```suggestion
给定此 CSP 标头：
```

File: files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md [hunk 4]
```diff
@@ -5,56 +5,74 @@ slug: Web/HTTP/Headers/Content-Security-Policy/worker-src
...
 {{Compat}}
 
-## See also
+## 参见
 
-- {{HTTPHeader("Content-Security-Policy")}}
-- [CSP for Web Workers](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers#Content_security_policy)
-- {{domxref("Worker")}}, {{domxref("SharedWorker")}}, {{domxref("ServiceWorker")}}
+- {{HTTPHeader("Content-Security-Policy","内容安全策略")}}
+- [CSP for Web Workers](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers#内容安全策略)
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/content-security-policy/worker-src/index.md:77]:
```suggestion
- {{HTTPHeader("Content-Security-Policy")}}
- [Web Worker 的内容安全策略（SCP）](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers#内容安全策略)
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

**Review by yin1999 — APPROVED** [review]

**Pull request merged**